R/geom-smooth.r

```diff
@@ -91,7 +91,7 @@ geom_smooth <- function(mapping = NULL, data = NULL,
     ...
   )
   if (identical(stat, "smooth")) {
-    params$method <- method
+    params$method <- if (!is.null(method) && identical(method, "auto")) NULL else method
```

Comment:
This check should be handled lower down so you don't need to repeat it in both `geom_smooth()` and `stat_smooth()`